examples/install.yaml

```
@@ -6,14 +6,13 @@ metadata:
   labels:
     app: crossplane-provider-terraform
 spec:
-  image: crossplane/provider-terraform-controller:v0.2.0
   args: ["-d"]
 ---
 apiVersion: pkg.crossplane.io/v1
 kind: Provider
 metadata:
   name: crossplane-provider-terraform
 spec:
-  package: crossplane/provider-terraform:v0.2.0
+  package: crossplanecontrib/provider-terraform:v0.5.0
```

```suggestion
  package: crossplanecontrib/provider-terraform:v0.6.0
```
Sorry, last nit, let's include the latest release here